doc/python/hardware.rst

```diff
@@ -152,9 +152,8 @@ The Python wrapper provides an API for:
 
         .. attribute:: threads
 
-            The number of threads available locally for execution. Must be set to 1 at minimum. 1 by default.
-            Passing ``"avail_threads"`` (as string) will query and use the maximum number of threads the system makes available.
-
+            The number of threads available locally for execution. Defaults to the maximum number of threads the system makes available if gpu_id and mpi are not set, else defaults to 1.
```

Comment:
A quick word/investigation into whether this is honouring the CGroups/ExecMask settings would be nice, but not a blocker.

Reply:
It does (as per arbenv::thread_concurrency()).

Reply:
I know, but it would be nice to have it written in the docs.

Reply:
OK, it can be added.